### PR TITLE
Form: label template

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: js/ui/form/components/label.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import $ from '../../../core/renderer';
 import { isDefined } from '../../../core/utils/type';
-import { isEmpty } from '../../../core/utils/string';
+import { getPublicElement } from '../../../core/element';
+// import { isEmpty } from '../../../core/utils/string';
```
Comment (ksercs) [inline js/ui/form/components/label.js]: .

File: js/ui/form/components/label.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 import { getLabelMarkText } from '../ui.form.layout_manager.utils';
 
 import {
-    WIDGET_CLASS,
+    // WIDGET_CLASS,
```
Comment (ksercs) [inline js/ui/form/components/label.js]: .

File: testing/tests/DevExpress.ui.widgets.form/form.markup.tests.js [hunk 3]
```diff
@@ -1132,6 +1132,39 @@ QUnit.module('Grouping', () => {
...
+                    items: [
+                        {
+                            dataField: 'firstName',
+                            label: {
+                                template: (data, container) => {
+                                    assert.deepEqual(isRenderer(container), !!config().useJQuery, 'container is correct');
+
+                                    $('<div>')
+                                        .text(data.text + ' ?')
+                                        .addClass('label-template')
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.form/form.markup.tests.js]: let's move this string to a constant?

File: testing/tests/DevExpress.ui.widgets.form/form.markup.tests.js [hunk 4]
```diff
@@ -1132,6 +1132,39 @@ QUnit.module('Grouping', () => {
...
+                            }
+                        }
+                    ]
+                }]
+        });
+        const $groups = $formContainer.find(`.${FIELD_ITEM_CLASS}`);
+
+        assert.equal($groups.length, 2, '2 groups rendered');
+        assert.equal($groups.eq(0).find('.label-template').length, 1, 'label template content');
+        assert.equal($groups.eq(0).find('.label-template').text(), 'First Name: ?', 'Labels\'s content has correct data');
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.form/form.markup.tests.js]: Let's use `strictEqual` everywhere.
Let's mix different types of quotes to get rid of screen adaptation symbol

```suggestion
        assert.strictEqual($groups.eq(0).find('.label-template').text(), 'First Name: ?', "Labels's content has correct data");
```

File: testing/tests/DevExpress.ui.widgets.form/formLayoutManager.markup.tests.js [hunk 5]
```diff
@@ -2196,8 +2221,42 @@ QUnit.module('Templates', () => {
         assert.equal(textArea.option('value'), layoutManager.option('layoutData.test'), 'Widget\'s value equal to bound datafield');
     });
 
-    test('Check arguments of the template', function(assert) {
-        const templateStub = sinon.stub();
+    test('Render label template', function(assert) {
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.form/formLayoutManager.markup.tests.js:2224]: What the difference between this test and "Default render with label template" test?

File: testing/tests/DevExpress.ui.widgets.form/form.markup.tests.js (hunk 4 above)
Comment (EugeniyKiyashko) [reply testing/tests/DevExpress.ui.widgets.form/form.markup.tests.js]: Lint rule doesn't allow to do that

File: testing/tests/DevExpress.ui.widgets.form/formLayoutManager.markup.tests.js (hunk 5 above)
Comment (EugeniyKiyashko) [reply testing/tests/DevExpress.ui.widgets.form/formLayoutManager.markup.tests.js:2224]: https://github.com/DevExpress/DevExtreme/pull/22926/commits/97f177b700785f984d4f5858b0b0f633f127c7f4

File: js/ui/form/components/label.js (hunk 1 above)
Comment (EugeniyKiyashko) [reply js/ui/form/components/label.js]: https://github.com/DevExpress/DevExtreme/pull/22926/commits/97f177b700785f984d4f5858b0b0f633f127c7f4

**Review by IlyaKhD — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**